ui/store/actions.ts

```diff
@@ -5370,6 +5370,34 @@ export function fetchAndUpdateMetamaskNotifications(): ThunkAction<
   };
 }
 
+/**
+ * Synchronizes accounts data with user storage between devices.
+ *
+ * This function sends a request to the background script to sync accounts data and update the state accordingly.
+ * Upon success, it dispatches an action with type `SYNC_INTERNAL_ACCOUNTS_WITH_USER_STORAGE` to update the Redux state.
```

Comment:
Hmm no I don't think this is true.

@matteoscurati confirm? I think these are old comments that were never removed.

We can create a cleanup ticket.